packages/bpk-component-barchart/src/utils.js

```diff
@@ -16,24 +16,26 @@
  * limitations under the License.
  */
 
-const center = scale => {
+/* @flow strict */
+
+const center = (scale: Function) => {
```

Comment:
We typically try to avoid `Function` in favour of more specific syntax like `(argType) => returnType`, is there something more specific we can use here?